### ISSUE TITLE
The metric is not comparable

### ISSUE THREAD

**Issue by lampts** [bradleypallen/keras-quora-question-pairs#1]
Thank you for the work, and reference collection on this interesting dataset. As my understanding, these benchmarks are based on different test set. Secondly, the complexity of each solutions(e.g. number of parameters) is also a good indicator, for example the leaderboard from Stanford NLI here: http://nlp.stanford.edu/projects/snli/

My current solution:

Model params (693K), dataset: dev split: 0.1, test split: 0.1
loss      = 0.3608
accuracy  = 0.8336
precision = 0.7516
recall    = 0.8228
F         = 0.7782
CPU times: user 1min 36s, sys: 13.2 s, total: 1min 49s
Wall time: 1min 48s

**Comment by bradleypallen** [comment]
It is true that the accuracy estimates in the table are not based on identical test sets; the IBM Watson folks have I believe released their train/test split so that may be a basis for beginning to bring that kind of consistency into play. Am I missing what you're saying here?

**Comment by bradleypallen** [comment]
Closing as I believe the work in the context of the Kaggle competition has addressed this issue of getting a basis for a common test set. - cheers, BPA
